runtime/doc/eval.txt

```diff
@@ -4418,6 +4420,10 @@ jobsend({job}, {data})					{Nvim} *jobsend()*
 			:call jobsend(j, ["abc", "123\n456", ""])
 < 		will send "abc<NL>123<NUL>456<NL>".
 
+		If the job was started with the rpc option this function
+		cannot be used, instead use |rpcnotify()| and |rpcrequest()|
+		to communicate with the job.
```

Comment:
Doesn't need to block this PR, but this is a case where more generic language could help UX. We could:

- deprecate `jobsend`, `rpcnotify`, `rpcrequest`
- introduce `chansend({job/chan}, {raw=boolean}, {data})` and `channotify({job/chan}, ...)`

**Alternative suggestion:** Remove distinction between "rpc/channel" and "job". Just call everything a job.

- Jobs may have channels.
- Jobs that have channels work with a superset of `job*()` functions and parameters.